README.md

```diff
@@ -55,24 +41,30 @@ for us, which you can activate via::
 
   pipenv shell
 
-See the `pipenv documentation <https://docs.pipenv.org/>`_ for more detail.
+See the [pipenv documentation](https://pipenv.pypa.io/en/latest/) for more detail.
+
+If you prefer to use pip directly in your venv, specify the following
+requirements files:
+
+  - requirements.txt
+  - test-requirements.txt
+
+There is also a dist-requirements.txt, if you will be building the project
```

Comment:
Should docs-requirements.txt be mentioned here or in the next section?

Reply:
Sure, I can add that